catroid/build.gradle

```diff
@@ -59,6 +60,7 @@ android {
             buildConfigField "boolean", "FEATURE_ARDUINO_ENABLED", "true"
             buildConfigField "boolean", "FEATURE_RASPI_ENABLED", "true"
             buildConfigField "boolean", "FEATURE_POCKETMUSIC_ENABLED", "true"
+            buildConfigField "boolean", "FIREBASE_CRASH_REPORT", "false"
```

Comment:
Is there a good reason that this flag should be named different than the others? If not, please change it to something like `FEATURE_FIREBASE_CRASH_REPORT`